code/game/objects/items/devices/pizza_bomb.dm

```diff
@@ -10,6 +10,7 @@
 	var/wires = list("orange", "green", "blue", "yellow", "aqua", "purple")
 	var/correct_wire
 	var/armer //Used for admin purposes
+	var/opener //Ditto
```

Comment:
these two are gonna cause GC failures. they need nulling out in a `Destroy` override. if you have any questions feel free to ping me

Reply:
Resolved in latest PR. Thank you for the help!